cmd/esc/cli/testdata/env-set-string.yaml

```diff
@@ -0,0 +1,10 @@
+run: |
+  esc env init default/test
+  esc env set default/test path "[a" --string
```

Comment:
This test should also run an `esc env get` to ensure that the string's content is as expected